### DISCUSSION TITLE
Better phase system

### DISCUSSION THREAD

**Discussion by LuluxClient** [BG-Software-LLC/SSB-OneBlock#143] [Q&A]
Currently to edit phases you needs to add 42343243 lines in a json file and it's a hellish mess and totally not moduable§. Also it would be cool to have a requirement system where if a players doesn't have X island level and / ore X permission and X money in the banque etc... even the possibility to pay to go to the next phase. Executed commands at the end and many more.

**Comment by LuluxClient** [comment]
Also a yaml file so it's easier to add things and to read the file. Feel free to check out BentoBox for that, it's the only good thing about it xd

**Comment by Lemvs** [comment]
@OmerBenGera can you move this to the OneBlock repository?
